src/index.js

```diff
@@ -111,59 +109,41 @@ export default class BroccoliDependencyFunnel extends Plugin {
       return;
     }
 
-    let rollupOptions = {
+    let options = {
       entry: this.entry,
       external: this.external || [],
-      dest: 'foo.js',
-      plugins: [
-        {
-          resolveId: function(importee, importer) {
-            let moduleName;
-
-            // This will only ever be the entry point.
-            if (!importer) {
-              moduleName = importee.replace(inputPath, '');
-              modules.push(moduleName);
-              return path.join(inputPath, importee);
-            }
-
-            // Link in the global paths.
-            moduleName = amdNameResolver(importee, importer).replace(inputPath, '').replace(/^\//, '');
-            let modulePath = path.join(inputPath, moduleName + '.js');
-            if (existsSync(modulePath)) {
-              modules.push(moduleName + '.js');
-              return modulePath;
-            }
-          }
-        }
-      ]
+      dest: 'foo.js'
     };
 
-    let rollupNode = heimdall.start({
-      name: 'BroccoliDependencyFunnel (Rollup)'
+    let mrDepWalkNode = heimdall.start({
+      name: 'BroccoliDependencyFunnel (Mr Dep Walk)'
     });
-    return rollup(rollupOptions).then(() => {
-      stats.rollup++;
-      logger.debug('rollup executed');
 
-      this._depGraph = modules.sort();
-      this._nonDepGraph = filterDirectory(inputPath, '', function(module) {
-        return modules.indexOf(module) === -1;
-      }).sort();
+    modules = mrDepWalk.depFilesFromFile(this.inputPaths[0], options);
+    if (modules.indexOf(options.entry) === -1) {
+      modules.unshift(options.entry);
```

Comment:
Can you add a comment of what is going on here? I understand the code, but don't understand why it is needed.